pkg/dogstatsd/parser_test.go

```diff
@@ -204,64 +213,46 @@ func TestParseGaugeWithEmptyHostTag(t *testing.T) {
 	assert.InEpsilon(t, 1.0, parsed.SampleRate, epsilon)
 }
 
-func TestParseGaugeWithNoHostTag(t *testing.T) {
```

Comment:
removed this test as it's a duplicate of `TestParseGaugeWithTags` now that we check for the hostname in every test